content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -112,7 +112,7 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 4. If Authorization is checked in the wizard, only the sidecar is enabled. Refer to [Authorization](../../authorization/deployment/helm/) to install and configure the CSM Authorization Proxy Server.
 
-5. If Replication is checked in the wizard, only the sidecar is enabled. Refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters and the replication controller.
+5. If Replication is checked in the wizard, refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters.
```

Comment:
Refer link should be reverted to previous one.
refer to [Replication](../csmoperator/modules/replication) for the necessary prerequisites required for this module.

Reply:
Sorry, I don't get it. Would you mind modifying the file directly to what you expect ?

Reply:
Updated.